salt/utils/schedule.py

```diff
@@ -385,7 +385,7 @@ class Schedule(object):
     '''
     instance = None
 
-    def __new__(cls, opts, functions, returners=None, intervals=None, cleanup=None, proxy=None):
+    def __new__(cls, opts, functions, returners=None, intervals=None, cleanup=None, proxy=None, standalone=None):
```

Comment:
I think it would be more intuitive here if the default were `False` instead of `None`.